### PR TITLE
Fixes while testing recent products

### PR DESCRIPTION
**Pull request by mzieg**
- add ProcessedReading.to_dict
- default missing raman_intensity_calibration_order in Andor virtual EEPROM

### CODE REVIEW

File: wasatch/AndorDevice.py [hunk 1]
```diff
@@ -435,7 +435,7 @@ def update_config_from_eeprom(self, eeprom):
                 self.config_values[k] = v
 
     def _load_config_values(self):
```
Comment (samiebee43) [inline wasatch/AndorDevice.py:437]: This function is messy. I recommend a docstring:
```
"""
Loads configuration from file `self.config_file` and populates `self.settings.eeprom` with members.
"""
```

This falls under the category of scope manipulation, which is bad, but if we're going to do it lets be clear about it in the comments.

**Review by samiebee43 — CHANGES_REQUESTED** [review]

File: wasatch/AndorDevice.py (hunk 1 above)
Comment (mzieg) [reply wasatch/AndorDevice.py:450]: Documentation updated.

Comment (samiebee43) [reply wasatch/AndorDevice.py:450]: Thanks for adding a docstring to `update_config_from_eeprom`.

I was actually referring to the function `_load_config_values`.
I wrote in that docstring as a separate PR: https://github.com/WasatchPhotonics/Wasatch.PY/pull/95/files

---

When a click to add a comment to a function, github only shows the first line above my comment:
image

**Review by samiebee43 — APPROVED** [review]

**Comment by samiebee43** [comment]
What happened here? merge from CLI ?

**Comment by mzieg** [comment]
It was weird.  I did a merge from master using the website, but it wasn't showing the change on the website. I switched to the CLI and it seemed to work.  Looking at those two commits though, now I need to confirm.

**Pull request closed without merging**